syft/execution/placeholder.py

```diff
@@ -34,8 +34,6 @@ def instantiate(self, tensor):
         """
         if isinstance(tensor, PlaceHolder):
             self.child = tensor.child
-        elif tensor.is_wrapper:
-            self.instantiate(tensor.child)
```

Comment:
Removing this seems questionable, because we do want to be able to instantiate a `Placeholder` with a wrapper tensor.